great_expectations/execution_engine/execution_engine.py

```diff
@@ -310,7 +313,18 @@ def resolve_metrics(
 
         resolved_metrics: Dict[Tuple[str, str, str], Any] = {}
 
-        metric_fn_bundle = []
+        metric_fn_bundle: List[BundledMetricConfiguration] = []
+
+        metric_fn_type: MetricFunctionTypes
+        metric_class: "MetricProvider"  # noqa: F821
+        metric_fn: Any
```

Comment:
Same as above.
```suggestion
        metric_fn: Callable
```

Reply:
@cdkini It is not always `Callable` (sql metric implementations return a wide variety of `SQLAlchemy` methods).  Thanks.